src/py42/modules/alertrules.py

```diff
@@ -0,0 +1,37 @@
+class AlertRulesModule(object):
+    def __init__(self, microservice_client_factory, user_client):
+        self.microservice_client_factory = microservice_client_factory
+        self._user_client = user_client
+        self._alert_client = None
+
+    @property
+    def rules(self):
```

Comment:
Is this supposed be property or just private? `sdk.alertrules.rules` sounds unecessary.